Code/GraphMol/QueryBond.h

```diff
@@ -42,6 +42,18 @@ class RDKIT_GRAPHMOL_EXPORT QueryBond : public Bond {
       dp_query = nullptr;
     }
   }
+  QueryBond(QueryBond &&other) noexcept : Bond(std::move(other)) {
+    dp_query = std::move(other.dp_query);
+  }
+  QueryBond &operator=(QueryBond &&other) noexcept {
+    if (this == &other) {
+      return *this;
+    }
+    QueryBond::operator=(std::move(other));
```

Comment:
Is this a double copy?

Reply:
no, I believe this is the pattern one uses to call the base class operators:
https://isocpp.org/wiki/faq/assignment-operators#assignment-op-in-derived-class